graylog2-web-interface/src/logic/rest/FetchProvider.js

```diff
@@ -108,3 +144,22 @@ export default function fetch(method, url, body) {
   }
   return promise();
 }
+
+export function fetchPlainText(method, url, body) {
```

Comment:
I'm a bit unhappy about this code duplication but didn't want to refactor the whole `FetchProvider` to generically support something other than `application/json` in this PR as it's being used in quite a *few* places…